pkg/config/load.go

```diff
@@ -21,7 +21,15 @@ var ProwgenFile = ".config.prowgen"
 
 // Prowgen holds the information of the prowgen's configuration file.
 type Prowgen struct {
+	// Private indicates that the openshift-ci-operator's github credentials should
```

Comment:
I think I'd word this slightly differently:

```go
// Private indicates that generated jobs should be marked as hidden 
// from display in deck and that they should mount appropriate git credentials
// to clone the repository under test
```

Reply:
Done